int_experian/int_experian_sfra/cartridge/static/default/js/edqUtils.js

```diff
@@ -412,11 +413,14 @@ function edqSetGlobalIntuitiveConfiguration() {
 	* the new parameters that we're including, since the webpage doesn't do a refresh items we're not changing values 
 	* For more information see Bug #126164 */
 	$("script[src=\"" + edqGlobalIntuitiveUnicornJsPath + "\"]").remove();
-	$("<script>").attr({
-		src:edqGlobalIntuitiveUnicornJsPath,
-		integrity:"sha512-ooVQYWcrVoGAZXC+qPMJaFsEkLB82EsT42J+p0U5INWm+NHrN+XvnWsALGv440Zg3a/QsDJ2L9XR0592fkSBmA==",
-		crossorigin:"anonymous"
-	}).appendTo("footer");
+	if (edqGlobalIntuitiveStagging)
+		$("<script>").attr({
+			src:edqGlobalIntuitiveUnicornJsPath,
+			integrity:"sha512-ooVQYWcrVoGAZXC+qPMJaFsEkLB82EsT42J+p0U5INWm+NHrN+XvnWsALGv440Zg3a/QsDJ2L9XR0592fkSBmA==",
+			crossorigin:"anonymous"
+		}).appendTo("footer");
+	else
+		$("<script>").attr("src", edqGlobalIntuitiveUnicornJsPath).appendTo("footer");
```

Comment:
Remove these changes from this pull request.

Reply:
ayyyy jose !